osx-manual-setup.md

```diff
@@ -172,13 +172,23 @@ bundle install
 > ld: library not found for -lssl
 > ```
 >
-> Try to fix it adding the following config:
+> Try to install the gem manually with the following config:
 >
 > ```sh
-> bundle config --local build.mysql2 --with-ldflags="-L$(brew --prefix openssl)/lib" --with-cppflags="-I$(brew --prefix openssl)/include"
+> gem install mysql2 -- --with-ldflags="-L$(brew --prefix openssl)/lib" --with-cppflags="-I$(brew --prefix openssl)/include"
```

Comment:
The issue with this is that the gem version may not match what we have in `Gemfile.lock`. I wonder why bundler approach stopped working. But either way, maybe specify also that the gem version must match `Gemfile.lock`.